kuma/javascript/src/toc.jsx

```diff
@@ -48,11 +62,20 @@ export default function TOC({ html }: Props) {
 
     return (
         <aside className="document-toc-container">
-            <section className="document-toc" ref={tocRef}>
+            <section className="document-toc" ref={documentTOCRef}>
```

Comment:
The React-way of doing this would be to use React state and then reflect that state in `className={...}`